### PR TITLE
HP-2174 frontpage updates

### PR DESCRIPTION
**Pull request by Riippi**
- Update the front page to look more like in the figma design
- Use HDS helsinki logo
- Switch header language menu from simple to normal (better mobile mode)

https://helsinkisolutionoffice.atlassian.net/browse/HP-2174

- Footer helsinki logo links now to sv/fi/en version depending on the selected language

https://helsinkisolutionoffice.atlassian.net/browse/HP-2060

### CODE REVIEW

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr316-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/common/header/Header.tsx [hunk 1]
```diff
@@ -71,6 +72,13 @@ function Header(): React.ReactElement {
     }
   };
 
+  // List all languages as primary languages
+  const sortLanguageOptions: LanguageSelectorProps['sortLanguageOptions'] = (
+    options: LanguageOption[],
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _activeLanguage
```
Comment (mikkojamG) [inline src/common/header/Header.tsx]: I think this can be happily left out since it is not used? 🤔

**Review by mikkojamG — APPROVED** [review]
One small comment, otherwise LGTM

File: src/common/header/Header.tsx (hunk 1 above)
Comment (Riippi) [reply src/common/header/Header.tsx]: yes, fix:ed

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr316-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr316-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**